src/frontend/src/components/loader.ts

```diff
@@ -33,8 +40,11 @@ const startLoader = () => {
   };
 };
 
-export const withLoader = async <A>(action: () => Promise<A>): Promise<A> => {
-  const endLoader = startLoader();
+export const withLoader = async <A>(
+  action: () => Promise<A>,
+  showCheckOngoingIssues?: boolean
```

Comment:
Where do you pass this? I don't see it used in any showcase page nor anywhere else...

Reply:
It's used in the `loaderTakingForever.astro` file, to avoid having to wait the 10 seconds, there's two loading screen screenshots, one with button and one without.

Reply:
Oh, sorry. The `true` was on the other line in my screen and I missed it